src/docs/concepts-payments/provider-onboarding-and-identity-verification/index.md

```diff
@@ -194,61 +204,71 @@ after the account has been created.
+strongly recommend that you always use Stripe Connect Onboarding to
+onboard your providers, regardless of your front-end application.
 
 In our older
 [legacy templates](/how-to/provider-onboarding-and-identity-verification/#using-deprecated-payoutdetailsform-and-payoutdetailspage-as-a-starting-point),
-Stripe onboarding was implemented like this. There are some now
+Stripe onboarding was implemented with a custom flow. There are some now
 deprecated components you can use as a starting point if you want to
 implement your own flow. You should keep in mind that these components
 will not be updated by our team since Sharetribe Web Template uses
 Connect Onboarding by default.
 
-### Collecting required information
-
-When onboarding Custom accounts, you need to collect the required
-information for each account (which Stripe verifies). The information
-you need to collect depends on factors such as the capabilities and the
-country the account is based in. You can find the requirements from
-Stripe's
-[Required verification information](https://stripe.com/docs/connect/required-verification-information)
-documentation. You should keep in mind that depending on the situation,
-you might need to collect a scan of an ID document, an address document,
-or both to enable payouts so you may need to implement a way to upload
-these.
-
-After you have collected the information you need to obtain a new
-[account token](https://stripe.com/docs/connect/account-tokens) via
-Stripe API, through the same Stripe platform account as the one used by
-the marketplace. This account token should be passed to Sharetribe API
-when creating or updating the Stripe Account.
-
-### Updating the Stripe Account
-
-If you decide to use your own custom flow, you should concider
-implementing a way to check the status of a Stripe Account and collect
-more information about the user when it's required. Otherwise, the
-Stripe Account might get restricted and payouts to the provider will
-fail.
-
-### Using deprecated PayoutDetailsForm and PayoutDetailsPage as a starting point
-
-We have deprecated the old `PayoutDetailsForm` and
-`PayoutPreferencesPage` where the custom flow for onboarding was
-implemented. Also, functions related to the deprecated form have been
-removed from `stripe.duck.js`. If you want to keep using the custom form
-in your own application, you can take these deprecated components as a
-starting point but you should keep in mind that **they will not be
-updated** by our team.
-
 You can find the deprecated files still from v.3.7.0
 
 - [PayoutDetailsForm](https://github.com/sharetribe/ftw-daily/tree/v3.7.0/src/forms/PayoutDetailsForm)
 - [PayoutPreferencesPage](https://github.com/sharetribe/ftw-daily/tree/v3.7.0/src/containers/PayoutPreferencesPage)
 - [stripe.duck.js](https://github.com/sharetribe/ftw-daily/blob/v3.7.0/src/ducks/stripe.duck.js).
+
+## Stripe for customers: saved payment methods
```

Comment:
We have this article: https://www.sharetribe.com/docs/how-to/save-payment-card
on how the template handles saved payment method. So maybe this section could be client agnostic

Reply:
Also we could mention what relevance setupIntents have in this process: https://www.sharetribe.com/api-reference/marketplace.html#create-stripe-setup-intents